Source/Lib/Codec/EbEncHandle.c

```diff
@@ -4049,7 +4079,8 @@ EB_ERRORTYPE EbOutputBufferHeaderCtor(
     EB_PTR objectInitDataPtr)
 {
     EB_H265_ENC_CONFIGURATION   * config = (EB_H265_ENC_CONFIGURATION*)objectInitDataPtr;
-    EB_U32 nStride = (EB_U32)(EB_OUTPUTSTREAMBUFFERSIZE_MACRO(config->sourceWidth * config->sourceHeight));  //TBC
+    EB_U32 nStride = outputStreamBufferSize(config->sourceWidth, config->sourceHeight,
+            config->encoderColorFormat, config->encoderBitDepth, config->compressedTenBitFormat);
```

Comment:
Hi Austin, is your new logic takes care of this work around below? If yes, could you clean up?
```    
if (config->rateControlMode == 1 && config->targetBitRate >= 50000000) {
        nStride = 10000000;	        nStride = 10000000;
 }
```

Reply:
Cleaned up with the force-pushed commit, by which 8K input was also verified.